### PR TITLE
Removes `*/` from block doc comments

### PR DESCRIPTION
**Pull request by kjeremy**
The trailing `/` was annoying me on hover.

### CODE REVIEW

**Comment by kjeremy** [comment]
An alternate implementation would be to provide a `suffix: Option` similar to the `prefix` but I didn't think that would buy us much. I can go down that route though if people like it better.

File: crates/ra_syntax/src/ast/traits.rs [hunk 1]
```diff
@@ -136,7 +136,18 @@ pub trait DocCommentsOwner: AstNode {
                         prefix_len
                     };
 
-                line[pos..].to_owned()
+                let end = if comment.kind().shape.is_block() && line.ends_with("*/") {
+                    // FIXME: Use `nth_back` here once stable
+                    if line.chars().rev().nth(2).map(|c| c.is_whitespace()).unwrap_or(false) {
```
Comment (matklad) [inline crates/ra_syntax/src/ast/traits.rs]: Let's check that `c` s is ascii as well, otherwise this will panic on weird uicode whitespace characters.

Comment (bjorn3) [reply crates/ra_syntax/src/ast/traits.rs]: Maybe use `line[..line.len() - 2].trim_end()` below instead.

Comment (kjeremy) [reply crates/ra_syntax/src/ast/traits.rs]: I like the `trim_end()` on every comment line. Is there a reason we can't unconditionally do that?

Comment (kjeremy) [reply crates/ra_syntax/src/ast/traits.rs]: I just did that in the latest commit.

**Comment by kjeremy** [comment]
@matklad is this version good to go?

**Comment by matklad** [comment]
yep!

bors r+

**Pull request merged**